packages/babel-core/src/config/files/module-types.ts

```diff
@@ -68,12 +68,12 @@ function loadCtsDefault(filepath: string) {
       configFile: false,
       sourceType: "script",
```

Comment:
I think `.cts` files are supposed to be parsed as `"unambiguous"`. If I try this CTS file:
```js
export {};
with ({});
```

TS complains that `with` cannot be used in strict mode, so it's parsing it as a module.